torq/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: torq_web_1
+      APP_PORT: 7028
+  
+  web:
+    image: lncapital/torq:0.10.3@sha256:cacd93adc606f9e1e6740c9402a00caeb9490971b7715b8d1316a66a1af12749
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_LIGHTNING_NODE_DATA_DIR}:/lnd:ro
+    command:
+      - --torq.port
+      - "7028"
+      - --db.host
+      - torq_db_1
+      - --db.password
+      - moneyprintergobrrr
+      - --torq.password
+      - ${APP_PASSWORD}
```

Comment:
Is it possible to remove the password authentication? Umbrel automatically protects apps from unauthorised access using the user's credentials (via the App Proxy).

Reply:
As it stands it's not possible for us to run passwordless but we will add this in shortly for a nicer experience on Umbrel. Would be be ok to send a PR in later removing password requirement?

Reply:
Yes absolutely 👍